queries.sql

```diff
@@ -61,4 +61,114 @@ SELECT COUNT(neutered) FROM animal
 WHERE neutered = true;
 
 SELECT COUNT(neutered) FROM animal
-WHERE neutered = false;
\ No newline at end of file
+WHERE neutered = false;
+
+-- third pull reques
+UPDATE animal
+SET species_id = (
+  SELECT id
+  FROM species
+  WHERE species.name_= 'Digimon'
+  LIMIT 1
+)
+WHERE animal_name LIKE '%mon';
+
+UPDATE animal
+SET species_id = (
+  SELECT id
+  FROM species
+  WHERE species.name = 'Pokemon'
+)
+WHERE animal.name_ NOT LIKE '%mon';
+
+Sam Smith owns Agumon.
+
+UPDATE animal
+SET owner_id = (
+  SELECT id
```

Comment:
- [ ] All the insertions and modifications happen in the `data.sql` file. Kindly move them there 👍 

### Instructions:
![image](https://github.com/amanuelbelete95/vet-clinic-database/assets/50990925/7a368021-3d05-49e7-959e-8563b3f7eb41)